### DISCUSSION TITLE
Is there an authorization endpoint on postal server that can be used by NginX to authorize mail requests?

### DISCUSSION THREAD

**Discussion by ghevge** [postalserver/postal#2577] [Help with using Postal]
I'm trying to set up NginX as a mail proxy for postal (https://docs.nginx.com/nginx/admin-guide/mail-proxy/mail-proxy/), but it seems that it will require an authorization http enpoint, something similar to this:
https://github.com/reinvented-stuff/nginx-mail-auth-http-server

Does postal offer such a service?

Thanks

**Comment by willpower232** [comment]
I don't think Postals HTTP API would conform to nginx's expectations without some reworking.

What are you looking to accomplish by introducing a proxy like this?
